openfe/protocols/openmm_utils/omm_settings.py

```diff
@@ -332,6 +335,15 @@ class OpenMMEngineSettings(SettingsBaseModel):
     Default ``None``.
     """
 
+    @field_validator('compute_platform')
+    def supported_sampler(cls, v):
+        supported = ['cpu', 'opencl', 'cuda']
```

Comment:
Just adding a link here so if we ever decide we need HIP support, I can find this easily 

https://github.com/OpenFreeEnergy/openfe/pull/951